contracts/RestrictedNORI.sol

```diff
@@ -400,12 +389,10 @@ contract RestrictedNORI is
     if (!schedule.exists) {
       revert NonexistentSchedule({scheduleId: scheduleId});
     }
+    uint256 totalSupply = totalSupply(scheduleId);
     return
-      _scheduleTrueTotal(schedule.totalClaimedAmount, scheduleId) -
-      _releasedBalanceOfSingleSchedule(
-        scheduleId,
-        schedule.releasedAmountFloor
-      );
+      schedule._scheduleTrueTotal(totalSupply) -
+      schedule._releasedBalanceOfSingleSchedule(totalSupply);
```

Comment:
should probably just add another library function for this

Reply:
already part of the lib

Reply:
I see what your'e saying now haha.  added library functions `_claimableBalanceForSchedule`, `_claimableBalanceForScheduleForAccount`, and `_revocableQuantityForSchedule`